src/Microsoft.Graph/Exceptions/GeneratedErrorConstants.cs

```diff
@@ -4,16 +4,16 @@
 
 namespace Microsoft.Graph
```

Comment:
First of all locatilization in my opinion is wrong (src/Microsoft.Graph/Exceptions/GeneratedErrorConstants.cs). If you want to sperate that to diffrent module and follow modular monolith architecture then I think we should move whole repository code into that localization and leave only interface as a part of our domain contract.
Secondly I don't think having that in the namespace Microsoft.Graph is a good idea. I prefer have NewOrbit prefix for our libraries